cli/command/context/list.go

```diff
@@ -61,39 +60,19 @@ func runList(dockerCli command.Cli, opts *listOptions) error {
 		if kubernetesEndpoint != nil {
 			kubEndpointText = fmt.Sprintf("%s (%s)", kubernetesEndpoint.Host, kubernetesEndpoint.DefaultNamespace)
 		}
+		if rawMeta.Name == command.DefaultContextName {
```

Comment:
If `dockerCli.ContextStore().ListContexts()` is handled by `ContextStoreWithDefault.ListContexts ()`, is this special case still needed, or would it already return the right information?

Reply:
@thaJeztah I didn't want to add specific code for import/export command to avoid, when importing an exported default context, having a second context with the "Current DOCKER_HOST based configuration" description. Therefore I decided to leave the description empty for the default context and add handle that case here.

Reply:
Still a bit on the fence on this one (i.e., the same applies to other contexts, which also by default copy the description), but not a blocker

Reply:
Indeed. I think for other contexts it is less confusing. Maybe it would be nice to add a `--description` flag to import command in order to override the imported context description